modules/data-entry/src/main/frontend/src/vocabQuery/BrowseVocabularyBranch.jsx

```diff
@@ -216,10 +216,10 @@ function ListChild(props) {
   );
 }
 
-ListChild.propTypes = {
+VocabularyBranch.propTypes = {
     classes: PropTypes.object.isRequired
 };
 
-const BrowseListChild = withStyles(BrowseTheme)(ListChild);
+const BrowseVocabularyBranch = withStyles(BrowseTheme)(VocabularyBranch);
```

Comment:
Don't rename for export
```suggestion
export default withStyles(BrowseTheme)(VocabularyBranch);
```